src/server/ConsoleShimPolicy.cpp

```diff
@@ -41,19 +22,36 @@ ConsoleShimPolicy ConsoleShimPolicy::s_CreateInstance(const HANDLE hProcess)
         // Convert to lower case, just in case
         std::transform(clientName.begin(), clientName.end(), clientName.begin(), std::towlower);
 
-        isCmd = clientName.compare(L"cmd.exe") == 0;
+        _isCmd = clientName.compare(L"cmd.exe") == 0;

```

Comment:
fwiw it seems like it would be reasonable to not check if it's pwsh _if it's cmd_, but... eh. whatever. it was always this way.